GliaWidgets/Coordinator/Engagement/Alert/Properties/Item/AlertAction.swift

```diff
@@ -0,0 +1,11 @@
+struct AlertAction {
+    enum Style {
+        case regular
+        case destructive
+    }
+
+    let title: String
+    let style: Style
+
+    var handler: (() -> Void)?
```

Comment:
No, but this allows us to handle it straight in coordinator instead of referencing back the whole way through vm + delegates. Or do you have something else in mind?